File: MPI/mpicheck/tests/functions/1.0/MPI_Graphdims_get.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       INTEGER var_1
       INTEGER var_2
       INTEGER var_3
        call mpi_graphdims_get(var_0, var_1, var_2, var_3)
       call pmpi_graphdims_get(var_0, var_1, var_2, var_3)
        end program main
    

File: MPI/mpicheck/tests/functions/1.0/MPI_Graphdims_get.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Comm var_0;
    int *var_1;
    int *var_2;
    int ret;
    /* calls */
    ret = MPI_Graphdims_get(var_0, var_1, var_2);
    ret = PMPI_Graphdims_get(var_0, var_1, var_2);
    return 0;
}
